### DISCUSSION TITLE
Scene-Viewer making opaque material transparent

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4694] [General]
I noticed there are new instructions in Scene-Viewer AR, so I guessed it has been updated, but it started showing my model (which is solid/opaque white) as transparent, kind of foggy transparent. 

The model was exported from a three.js scene but it looks ok to me, also checked in the MV Editor and everything is ok with it also. 

After opening the model in blender I saw there was a node added - "gltf material output", which I didn't add. After deleting it and reexporting the model I saw that the node was added again. Only when I changed the specular component from 0.4 to 0.5 did it stop adding the node, once the node was gone the model worked as expected and was opaque once again. Not sure what was going on, hope someone here has the answers.

WebXR:
![Screenshot_20240304-112007_Chrome](https://github.com/google/model-viewer/assets/92090684/48e4228f-2cc6-4457-8eae-ac48433fe99c)

Scene-Viewer:
![IMG_20240304_112104](https://github.com/google/model-viewer/assets/92090684/b18da0bb-9fb9-4cd9-9e3a-94faa22cd800)

The added node:
![image_2024_03_04T10_43_57_912Z](https://github.com/google/model-viewer/assets/92090684/5b25706a-8264-46eb-bb7f-7d01cd91a2b8)

**Comment by danilokrulj** [comment]
@elalish Have you seen something like this happen before?

**Reply by elalish** [reply]
We don't control SV's rendering, so there are a variety of differences - hence why we recommend WebXR. I don't see anything in your screenshots that looks transparent. I would generally avoid "Specular" since that's not part of the normal metallic-roughness workflow and shouldn't be needed for most materials.
